platform/jobs/edxEndToEndRepoMasterWeeklyJob.groovy

```diff
@@ -0,0 +1,165 @@
+package devops
+
+import org.yaml.snakeyaml.Yaml
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_LOG_ROTATOR
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_GITHUB_BASEURL
+
+
+/*
+Example secret YAML file used by this script
+
+    authUser : name-of-auth-user-for-stage
+    authPass : password-of-auth-user-for-stage
+    loginEmail : email-for-stage-login
+    loginPass : password-for-stage-login
+    courseOrg: organization-of-course-for-test
+    courseNumber: course-number-of-course-for-test
+    courseRun: course-run-number-of-course-for-test
+    courseDisp: course-display-name-of-course-for-test
```

Comment:
Can you remove these lines now that they are not contained within a secret. It will make it easier for others to understand